### ISSUE TITLE
[Feature Request]: try to use turbo for our monorepos

### ISSUE THREAD

**Issue by alexander-akait** [cssnano/cssnano#1275]
### What should be improved?

build/lint/test pipeline, we don't replace pnpm, we just speedup our tasks

### Describe the solution you would like

https://turborepo.org/

### Additional context

No

### Are you willing to work on this?

- [ ] Yes, I would like to help

**Comment by ludofischer** [comment]
We could try, but we might end up not needing it for long. If we merge cssnano plugins the monorepo would become smaller.

**Comment by alexander-akait** [comment]
Yep, but maybe useful for build site and cssnano, maybe we still have 2-3 packages in monorepo, not high priority

**Comment by ludofischer** [comment]
I don't think we can benefit from Turborepo at the moment, because we've already removed the build step, so there's nothing to cache, and individual packages don't have separate test or lint tasks, so they can't run in parallel.

**Comment by ludofischer** [comment]
> I don't think we can benefit from Turborepo at the moment, because we've already removed the build step, so there's nothing to cache, and individual packages don't have separate test or lint tasks, so they can't run in parallel.

Closing as the older comment is still accurate.
